contact.html

```diff
@@ -14,15 +14,15 @@
 	
 	<body>
 	
-	<body background = "images/slide1.jpg" alt="large herd of zebras">	
+	<body background = "images/slide16.jpg" alt="large herd of zebras">	

```

Comment:
I dont seem to like both images very much I may have to find a different image for it